mysqltuner.pl

```diff
@@ -1,4 +1,4 @@
-#!env perl
+#!/usr/bin/perl
```

Comment:
This would be a bit better
```suggestion
#!/usr/bin/env perl
```

Reply:
relates to https://github.com/major/MySQLTuner-perl/pull/762